src/types/tracker.ts

```diff
@@ -21,6 +21,7 @@ type MappedEventNames =
 
 export type TrackedEventNames =
   | MappedEventNames
+  | 'SDK_FLOW_START'
```

Comment:
Is there any particular reason for this ununiform event name?

Reply:
I don't feel particularly strongly about this for now, so happy to change to be in line with the rest. 

But one thing that was clear when I did a run through our analytic events' format with mobile SDK devs is that our event name formatting is inconsistent + some are different from mobile SDKs, i.e. the `TrackedEventNames`. Was thinking to start with this one.

Reply:
This will add yet another inconsistency, therefore I would keep this entry consistent with the ones below and address the naming inconsistency in a separate story (probably part of the analytics work that Anurag is doing).